airsonic-main/src/main/java/org/airsonic/player/service/metadata/DefaultMetaDataParser.java

```diff
@@ -32,6 +33,7 @@
  * @author Sindre Mehus
  */
 @Service
+@Order(1000)
```

Comment:
Can you please explain (maybe in the commit message) why this one is 1000, and other is 100 and an other other is 0 ?

Reply:
Just to give a bit of space between to allow for extensibility.

Reply:
Basically we can add more additional MetaDataParsers and be able to order them between (or before or after) any of the existing ones.

Reply:
Maybe 0,100,200 makes more sense? Why 1000? Do we expect more parsers between FFMPEG and Default?

Also for additional consideration:
- If additional parsers are added, there are code changes anyway, and the authors can reorder again as they like. People will need to reorder some stuff with more and more parsers anyway. Consider:
  - Author 1 needs to add a Parser1 after FFMPEG and so he naturally uses 101.
  - Author 2 comes along and wants to add the next parser after FFMPEG and before Parser1, he'll _have_ to change order of either FFMPEG or Parser1, Either way it's a code change in another Parser.

Keeping the order linear makes it apparent what is going on.

Reply:
Sure I'll change it to be linear.